Documents/Install.md

```
@@ -70,17 +70,16 @@ It should show all the tests are passing.
 
 ### Building from Windows
 
-To build the code on Windows, CMake, Python, and Visual Studio 2017 (or higher) is required. Windows' version of CMake is available from [this website](https://cmake.org/), Python installer can be downloaded from [here](https://python.org/). For Python, version 2.7.9 or later is recommended. To install Visual Studio, the community edition of the tool can be downloaded from [Visual Studio Community 2017](https://www.Visualstudio.com/en-us/products/Visual-studio-community-vs.aspx).
+To build the code on Windows, install CMake, Python, and a current Visual Studio release with C++23 support.
 
 Once everything is installed, run the following commands:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)